modules/Microsoft.Compute/virtualMachines/deploy.bicep

```diff
@@ -98,10 +98,10 @@ param bootDiagnosticStorageAccountName string = ''
 param bootDiagnosticStorageAccountUri string = '.blob.${environment().suffixes.storage}/'
 
 @description('Optional. Resource name of a proximity placement group.')
-param proximityPlacementGroupName string = ''
+param proximityPlacementGroupResourceId string = ''
```

Comment:
VMSS should also probably be aligned to the same change

Reply:
done. Removed the ProximityGroup deployment & expect resource ID instead